packages/web/src/modules/strapi/types/components.d.ts

```
@@ -133,7 +133,7 @@ export interface SurveyGroup extends Struct.ComponentSchema {
 }
 
 declare module "@strapi/types" {
-  export namespace Public {
+  export module Public {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)